elasticizer/pipelines/extract_postgres.py

```diff
@@ -50,8 +50,14 @@ class Format(luigi.Task):
 
     def _fields_from_mapping(self):
         with open(self.mapping_file,'r') as fp:
-            mapping_json= json.load(fp,object_pairs_hook=collections.OrderedDict)
-        return mapping_json['properties'].keys()
+            mapping_json = json.load(fp, object_pairs_hook=collections.OrderedDict)
+            fields = mapping_json['properties'].keys()
+            # Automatically remove copy_to fields because they aren't expected from the input
+            copy_targets = set([i.get('copy_to','') for i in mapping_json['properties'].itervalues()])
```

Comment:
itervalues() is going away.  i think you want to use just `values()` going forward b/c that's what python 3 will be supporting, and they both return an iterator, i believe.

http://python3porting.com/differences.html